lib/terraforming/resource/route53_zone.rb

```diff
@@ -49,7 +50,9 @@ def tfstate
       private
 
       def hosted_zones
-        @client.list_hosted_zones.map(&:hosted_zones).flatten.map { |hosted_zone| @client.get_hosted_zone(id: hosted_zone.id) }
+        @client.list_hosted_zones.map(&:hosted_zones).flatten.select do |resource|
+          @match_regex ? module_name_of(resource) =~ @match_regex : 1
+        end.map { |hosted_zone| @client.get_hosted_zone(id: hosted_zone.id) }
```

Comment:
__[rubocop]__
* Avoid multi-line chains of blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

<!-- Comment by [SideCI](https://sideci.com) -->